ArduPlane/Parameters.h

```diff
@@ -401,6 +404,9 @@ class Parameters {
     AP_Int16 waypoint_radius;
     AP_Int16 waypoint_max_radius;
     AP_Int16 rtl_radius;
+    
+    // Thermalling
+    AP_Int16 loiter_radius;
```

Comment:
note that in parameter the macro is ASCALAR where other "local" params use GSCALAR. To access the ASCALAR values use aparm.loiter_radius instead of g.loiter_radius.